app/src/main/java/org/mozilla/focus/browser/LocalizedContent.kt

```diff
@@ -45,7 +45,7 @@ object LocalizedContent {
         substitutionMap["%about-version%"] = aboutVersion
         val aboutContent = resources.getString(R.string.about_content, appName, learnMoreURL)
         substitutionMap["%about-content%"] = aboutContent
-        val wordmark = HtmlLoader.loadPngAsDataURI(context, R.drawable.wordmark)
+        val wordmark = HtmlLoader.loadPngAsDataURI(context, R.drawable.mvp_wordmark)
```

Comment:
Naming nit: Once we've released, the term "MVP" will be redundant and we'd have to make changes around the code base to correct it after the fact. I know it's not ideal, but maybe `wordmark2`?
```suggestion
        val wordmark = HtmlLoader.loadPngAsDataURI(context, R.drawable.wordmark2)
```